eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/ExtendedBitcoinClient.scala

```diff
@@ -322,7 +336,9 @@ object ExtendedBitcoinClient {
    */
   case class MempoolTx(txid: ByteVector32, vsize: Long, weight: Long, replaceable: Boolean, fees: Satoshi, ancestorCount: Int, ancestorFees: Satoshi, descendantCount: Int, descendantFees: Satoshi)
 
-  case class Utxo(txid: ByteVector32, vout: Long)
+  case class UnlockOutpointUtxo(txid: ByteVector32, vout: Long)
```

Comment:
nit: I find it less confusing if that doesn't even mention "utxo" (it's actually just an outpoint, where we use `txid` instead of the reversed `hash`)

```suggestion
  case class UnlockOutpoint(txid: ByteVector32, vout: Long)
```

Reply:
Yes I wasn't satisfied either. Another idea was to put argument-specific classes inside a dedicated `Args` object.

Reply:
Done with 2400e7e.